src/main/java/elucent/albedo/lighting/LightManager.java

```diff
@@ -74,16 +76,38 @@ public static void update(World world) {
 			if (e instanceof ILightProvider){
 				addLight(((ILightProvider)e).provideLight());
 			}
+			if (e.hasCapability(Albedo.LIGHT_CAP, null)){
```

Comment:
I didn't see that issue before now. I understood it to be slower, but was not entirely aware of just how much so.